researcher.lic

```diff
@@ -0,0 +1,180 @@
+=begin
+  Documentation: https://elanthipedia.play.net/Lich_script_repository#researcher
+=end
+
+custom_require.call(%w[common common-arcana drinfomon events])
+
+
```

Comment:
🧹 Deleting extra blank line
```suggestion
```

Reply:
@Valherun this comment is marked resolved but the extra new line still appears in the file. Perhaps the commit hasn't been pushed? Thanks